cmd/minishift/cmd/delete.go

```diff
@@ -113,6 +114,16 @@ func removeInstanceConfigs() {
 	}
 }
 
+func removeProfileFromConfig() {
+	name := constants.GetMachineName()
```

Comment:
_constants.GetMachineName()_, I don't think _constant.go_ is a good place to add this. After all it is not constant.